### DISCUSSION TITLE
Should time be converted to UTC before saving the record?

### DISCUSSION THREAD

**Discussion by CovertCode** [pocketbase/pocketbase#5578] [Q&A]
In my collection I have a specific field `start_time`. My user will enter time according to their timezone. So should I convert the time in browser itself from X timezone to UTC then save it?
The local time we see in pb dashboard UI, is it stored in database or this conversion happens only in dashboard by take browser timezone?

**Comment by ganigeorgiev** [comment]
All PocketBase dates are stored and returned in UTC following the format `Y-m-d H:i:s.uZ` (the `Z` indicates UTC), so ideally the submitted date should match.

In case a different date layout is submitted we'll try to cast it to the above one if it matches some of the [common formats](https://github.com/spf13/cast/blob/master/caste.go#L1422), otherwise it will result to a zero `types.DateTime`, aka. empty string.

The date that we show in the UI on hover is generated on the client-side by converting the returned UTC record date to the current browser local timezone.
